### PR TITLE
Practice Disabler

### PR DESCRIPTION
**Pull request by Cheackraze**
## About the PR 
Makes a new practice disabler to go along with all the practice ammo for the guns. Its use case will mostly be fluff and admemes, since you can VV the bullet component on other guns into the new BulletDisablerPractice prototype to basically make a disabler useless.  The disabler bolt does 5 stamina damage and 1 heat damage, similar to how other practice rounds just kind of tickle.  With those values, someone wielding 2 disablers, who manages to hit all 20 shots on a target can still successfully disable somebody, although somebody robust enough to hit all 20 shots in a row with two disablers is just showing off anyways.
**Screenshots**

practicedisabler

**Changelog**

:cl: checkraze
- add: Added a practice disabler

### CODE REVIEW

**Comment by Cheackraze** [comment]
wait hold on now suddenly the inhands isn't working

**Comment by Cheackraze** [comment]
ok this might be an unrelated thing because regular disabler inhands also isn't working, along with some other guns I'm going to investigate

**Comment by Cheackraze** [comment]
ok, after investigating, have disabler in-hands sprites _ever_ worked right? they don't work on live right now and it looks like they never did or would work, and would need remaking entirely from scratch.

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml [hunk 1]
```diff
@@ -289,6 +289,41 @@
...
+  name: practice disabler
+  parent: BaseWeaponBatterySmall
+  id: WeaponDisablerPractice
+  description: A self-defense weapon that exhausts organic targets, weakening them until they collapse. This one has been undertuned for cadets.
+  components:
+    - type: Tag
+      tags:
+        - Taser
+    - type: Sprite
+      sprite: Objects/Weapons/Guns/Battery/practicedisabler.rsi
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:302]: snakecase, also shouldn't this just inherit from WeaponDisabler and change the prototype?

Comment (Cheackraze) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml:312]: the existing naming convention in that folder used both snakecase and not, so I just guessed. I will rename it. The new prototype would only inherit 2 total components if reparented but I will do it real quick.

Comment (Cheackraze) [reply Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml]: it is done

**Pull request merged**